super-test.sh

```diff
@@ -97,6 +97,9 @@ while [ $# -gt 0 ]; do
       ;;
     clang* )
       export CXX=clang++${1#clang}
+      if [ "$1" -eq clang-9]
```

Comment:
I suppose this is because clang-5.0 didn't support this flag? Let's do `-ne clang-5.0` instead of `-eq clang-9` so we don't accidentally stop using this if/when we upgrade to newer clangs in CI.

Reply:
I think it started at clang-6 indeed